Formula/fn.rb

```diff
@@ -0,0 +1,25 @@
+class Fn < Formula
+  desc "CLI tool for the fn project."
```

Comment:
No final period. "Command-line" instead of "CLI".

Reply:
Done